<ui version="4.0" >
 <class>OutputSettings</class>
 <widget class="QWidget" name="OutputSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBoxIdleSeparatorLines" >
     <property name="title" >
      <string>Enable away/back separator lines</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QLineEdit" name="lineEditIdle" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="pushButtonIdleColor" >
        <property name="text" >
         <string>Pick a color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="labelIdleLineLength" >
        <property name="text" >
         <string>Line length:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="spinBoxIdleLineLength" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="labelIdlePeriodicString" >
        <property name="text" >
         <string>Periodic separator string:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="lineEditIdlePeriodicString" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>389</width>
       <height>150</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
